mirage/factories/file.js

```diff
@@ -19,6 +19,7 @@ export default Base.extend({
   isDynamicDone: faker.random.boolean(),
   isManualDone: faker.random.boolean(),
   isApiDone: faker.random.boolean(),
+  canGenerateReport: faker.random.boolean(),
```

Comment:
factory keys should be have underscores instead of camel case

Reply:
Done